build-scripts/build_cpp_unit_tests.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Any command line arguments are assumed to be CMake arguments
+CMAKE_ARGS=$@
+
```

Comment:
We only have one target (for now) so it's not a huge deal, but could you nuke the build directory for the CPP unit tests as well?

Reply:
Mainly for consistency

Reply:
Ooop thanks for catching that. Added